openshift/generate-manifests.sh

```diff
@@ -70,6 +56,9 @@ done
   cd "$MANIFEST_DIR"
 
   for f in *; do
+    if [[ $f == *.md ]]; then 
```

Comment:
What .md files exist that need to be skipped?

Reply:
The README.md that I included in the rukpak manifests [here](https://github.com/operator-framework/rukpak/blob/main/manifests/README.md)